seqr/views/apis/users_api.py

```diff
@@ -36,6 +36,21 @@ def get_all_collaborators(request):
     return create_json_response(collaborators)
 
 
+@login_required(login_url=API_LOGIN_REQUIRED_URL)
+@csrf_exempt
+def get_all_analysts(request):
+    analysts = {}
+    for project in get_projects_user_can_view(request.user):
+        analysts.update(_get_project_collaborators(project, include_permissions=False))
```

Comment:
So this is very close to what you want, but with one small caveat: per the original ticket " When editing, you should be able to select any existing user with access to **the project** or any staff user."  So rather than looking at all collaborators who share a project with the current user, you will want to include the project_guid in the URL and only get collaborators for that project. Let me know if you'd like to discuss this one more in person